src/main/java/pl/piekoszek/gorskimatches/token/TokenCreator.java

```diff
@@ -0,0 +1,26 @@
+package pl.piekoszek.gorskimatches.token;
+
+import io.jsonwebtoken.JwtBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.UUID;
+
+@Component
+public class TokenCreator {
+
+    public static String jwtToken(String email) {
+
+        SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.HS256;
+        JwtBuilder tokenBuilder = Jwts.builder()
+                .claim("email", email)
+                .setId(UUID.randomUUID().toString())
+                .setIssuedAt(Date.from(Instant.now()))
+                .setExpiration(Date.from(Instant.now().plus(10l, ChronoUnit.MINUTES)));
```

Comment:
10 minutes is a bit small.
better use `10L`, not `10l` but in this case `10` is ok

Reply:
Changed to 15

Reply:
So we want user login every 15 minutes? I had in mind about a month.

Reply:
Right :D Good point :D